### DISCUSSION TITLE
Purge gone?

### DISCUSSION THREAD

**Discussion by Aromack** [uBlockOrigin/uAssets#22043] [Q&A] [answered]
After I got a youtube.com adblock ban (happens at least twice a week). I went through the process I usually go through. Click browser setting / history / clear recent history/  delete all cookies. Then I go into Ublock Origin / filters click purge then update now. But I noticed yesterday that the purge button is gone. Is there a reason for this? Was it removed because it is no longer necessary? 
![image](https://github.com/uBlockOrigin/uAssets/assets/108582813/ce385609-e8d9-4e1f-898f-3b1a095c8c67)

**Comment by gorhill** [comment, marked as answer]
> Was it removed because it is no longer necessary?

Yes, see <https://old.reddit.com/r/uBlockOrigin/comments/18t9ush/youtube_detection_ads_breakages_2023_12_29_ubo/kib7o8y/>:

**Reply by Aromack** [reply]
Cool, thank you very much for the info.

**Reply by garry-ut99** [reply]
Also more info here :
- https://github.com/uBlockOrigin/uAssets/issues/22028
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3081
